src/Core/Framework/Resources/views/documents/includes/position.html.twig

```diff
@@ -80,10 +80,29 @@ All blocks of this template are available in the template which renders this tem
                             <td class="align-right">{% block document_line_item_table_column_tax_rate %}{% for tax in lineItem.price.taxRules %}{{ tax.taxRate }} % {% if loop.last %}{% else %}<br>{% endif %}{% endfor %}{% endblock %}</td>
                         {% endblock %}
                         {% block document_line_item_row_table_unit_price %}
-                            <td class="align-right">{% block document_line_item_column_table_unit_price %}{{ lineItem.unitPrice|currency(currencyIsoCode, languageId) }}{% endblock %}</td>
+                            <td class="align-right">
+                                {% block document_line_item_column_table_unit_price %}
+                                    {% set unitPrice =  lineItem.unitPrice %}
+                                    {% if config.fileType == constant('Shopware\\Core\\Checkout\\Document\\Service\\HtmlRenderer::FILE_EXTENSION') %}
+                                        {% if unitPrice < 0 %}<span class="visually-hidden">minus</span><span aria-hidden="true">&minus;</span>{% endif %}{{ unitPrice|abs|currency(currencyIsoCode, languageId) }}
```

Comment:
Have you seen this comment in the original Jira issue?
https://shopware.atlassian.net/browse/NEXT-40922?focusedCommentId=543030

Did you check if using `&minus;` is already enough? That seems to be the solution the other team has merged and is much simpler than this, see:
https://gitlab.shopware.com/shopware/6/product/platform/-/merge_requests/15327/diffs#753348f66fa29156a451bb44ce98db879a9e1d09

Reply:
&minus; isnt enough. I read that there are different Screen readers. Some reeds (-), some reads (&minus;) and some nothing of them. Thats why i introduced it like i do. So, all can read the text and ignore the sing.

Reply:
Just talked with @tobiasberge about it. I think we should go with `&minus;` as that is also properly translated. Your `<span class="visually-hidden">minus</span>` solution will read out strangely in languages other than maybe German or English.

If it is still not working it may be because of the `currency` twig filter and the currency sign 🤔

Reply:
Alternatively, we can translate the minus

Reply:
There is no problem with the currency filter.. Thats a issue with some screen reader. You can find the problem if you google it.

Reply:
Which screen reader does not support this? For me (VoiceOver) reads the `&minus` just fine. The upside would be that we do not need an extra translation for this. Because it's implicitly translated by the screen reader itself.

Reply:
I read

https://jamescatt.ca/blog/2020/minus-the-minus-a-psa-about-screen-readers-and-negative-numbers/

https://www.deque.com/blog/dont-screen-readers-read-whats-screen-part-1-punctuation-typographic-symbols/

Its no problem for me to change it, because my knowledge only based on this two articles. But i want to mention that.

Reply:
Reduce it like discussed to the html entity